### PR TITLE
The informatica team added a  new step.  Deploy Multiple Deployment Groups.

### PR DESCRIPTION
**Pull request by leosraj**
This step will read a list of deployment groups from a file

### CODE REVIEW

**Comment by leosraj** [comment]
let me fix the conflicts locally and upload it again.

File: .gitignore [hunk 1]
```diff
@@ -14,3 +14,8 @@
 /releases
 /build
 /lib
+plugin/info.xml
```
Comment (nhmathis) [inline .gitignore]: Why are these files added to the .gitignore file? 

The updates to info.xml and plugin.xml will need to made too.

File: plugin/plugin.xml [hunk 2]
```diff
@@ -926,4 +926,97 @@
       <arg file="${PLUGIN_OUTPUT_PROPS}"/>
     </command>
   </step-type>
```
Comment (nhmathis) [inline plugin/plugin.xml:928]: Need to bump plugin.xml version to 28.

**Review by nhmathis — CHANGES_REQUESTED** [review]
If possible, try to format the new .groovy file. The spacing is inconsistent and makes it hard to read for future users.

File: .gitignore (hunk 1 above)
Comment (leosraj) [reply .gitignore]: We don't need anything in the .gitignore file. We will fix the info.xml and plugin. xml and get back to you

File: plugin/plugin.xml (hunk 2 above)
Comment (leosraj) [reply plugin/plugin.xml:928]: sure

**Review by nhmathis — APPROVED** [review]
